app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
```

Comment:
Automation does not require you throw an `MiqProvisionError` here, a raw provider exception will fail the automation.
Automation sometimes needs to catch specific exception, for example, it may need to get a status of an object. If the object does not exist in the provider, likely the API throws a provider specific object_not_found exception. Such exception should not fail the whole automation.
When a method is created in automation we try to make it re-useable across multiple providers. That's the reason we recommend to wrap provider specific exceptions to a miq exception. If we start to wrap some exceptions, it's better to wrap all exceptions even though they may not be required.
When wrapping an exception the original exception should not be consumed silently. Either we need to log it or embed it in the new exception.